### PR TITLE
Add release notes 60; other misc changes

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: release-notes/2020-60.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## Community Projects
+
+- Open sourced new `deferred-maintenance` CLI tool to help track "maintenance
+  debt" as Finding entities using the JupiterOne graph platform. 
+
+  <https://github.com/JupiterOne/deferred-maintenance>
+
+## Other Improvements and Bug Fixes
+
+- Fixed an issue with integration job container width were inconsistent.
```
Comment (henrymgarrett) [inline release-notes/2020-60.md:94]: were -> that was

**Review by henrymgarrett — APPROVED** [review]

File: release-notes/2020-60.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## New Features
+
+- New version of the **Policies** app is being rolled out across accounts. This
+  does not have much functional change, but behind the scenes it is a major
+  overhaul with significant improvements on performance, usability and
+  maintainability! This will allows us to add popular feature requests in the
+  very near future such as text search across all policy/procedure documents. 
+
+- Beta version of the shiny new **Visual Query Builder**! This allows you to
```
Comment (austinkelleher) [inline release-notes/2020-60.md:13]: Can we get a shiny gif of this in the release notes? I think adding some gifs/images adds tons of additional value to these!

**Comment by austinkelleher** [comment]
@fawcilize Worked on weekly alert reports

Screen Shot 2020-12-16 at 9 47 53 AM

File: release-notes/2020-60.md (hunk 2 above)
Comment (alee-j1) [reply release-notes/2020-60.md:13]: Shiny gif comin in hot, courtesy of @henrymgarrett!
shortWithRelationships

**Review by ndowmon — APPROVED** [review]

File: release-notes/2020-60.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  build J1QL queries by drag-n-drop and with guided directions. No coding or
+  querying knowledge required!
+
+<p align="center"><img src="../assets/visual-query-builder-WithRelationships.gif" alt="Visual Query Builder"></p>
```
Comment (erkangz) [inline release-notes/2020-60.md:17]:
```suggestion
![Visual Query Builder](../assets/visual-query-builder-WithRelationships.gif)```

File: release-notes/2020-60.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+- New version of the **Policies** app is being rolled out across accounts. This
+  does not have much functional change, but behind the scenes it is a major
+  overhaul with significant improvements on performance, usability and
+  maintainability! This will allows us to add popular feature requests in the
+  very near future such as text search across all policy/procedure documents. 
+
+- Beta version of the shiny new **Visual Query Builder**! This allows you to
+  build J1QL queries by drag-n-drop and with guided directions. No coding or
+  querying knowledge required!
```
Comment (alee-j1) [inline release-notes/2020-60.md:15]: How can users access the beta visual query builder?

Comment (henrymgarrett) [reply release-notes/2020-60.md:15]: I'm turning it on for everybody as we speak

File: release-notes/2020-60.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  >
+  > Please note currently queries via the API still use the legacy execution
+  > engine. We are rolling out this new engine across all accounts by EOY. This
+  > will be seamlessly deployed and all interfaces including UI and API will
+  > switch over. You can experience the power of this improvement sooner via this
+  > UI toggle.
+
+- Our integrations list continue to grow and it's getting a little hard to find
+  the specific one on the page, so we added the ability to filter the
+  integrations list by class/category.
```
Comment (alee-j1) [inline release-notes/2020-60.md:37]: Any screenshots for the new filtering capabilities?

**Pull request merged**